zip-0209.rst

```diff
@@ -0,0 +1,56 @@
+::
+
+  ZIP: 209
+  Title: Prohibit Negative Shielded Value Pool
+  Author: Sean Bowe <sean@z.cash>
+  Category: Consensus
+  Created: 2019-02-25
+  License: MIT
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+The term "block chain" and "network upgrade" are to be interpreted as defined in [#zip-0200]_.
+
+The "Sprout value pool balance" for a given block chain, as implied by section 4.11 of the Zcash Protocol Specification [#protocol]_,
+is the sum of all ``vpub_old`` fields for transactions in the block chain, minus the sum of all ``vpub_new`` fields for transactions
+in the block chain.
+
+The "Sapling value pool balance" for a given block chain, as implied by section 4.12 of Zcash Protocol Specification [#protocol]_,
+is the negation of the sum of all ``valueBalance`` fields for transactions in the block chain.
+
+The "Sapling value pool balance", as implied by section 4.12 of Zcash Protocol Specification [#protocol]_, is the negation of the sum of all previous transactions' ``valueBalance`` fields.
```

Comment:
I think this definition is unused in the text below (there's only one reference to "Sapling value pool balance" and it's to the "for a given blockchain" variant of the definition above).

Reply:
I think this is an artifact of accepting my suggestion, which was intended to replace the definition but seems to have kept the old one as well. It should just be removed.

Reply:
```suggestion
```